pkg/controller/datavolume/external-population-controller.go

```diff
@@ -84,16 +84,18 @@ func NewPopulatorController(ctx context.Context, mgr manager.Manager, log logr.L
 }
 
 // Reconcile loop for externally populated DataVolumes
-func (r PopulatorReconciler) Reconcile(ctx context.Context, req reconcile.Request) (reconcile.Result, error) {
+func (r *PopulatorReconciler) Reconcile(ctx context.Context, req reconcile.Request) (reconcile.Result, error) {
 	log := r.log.WithValues("DataVolume", req.NamespacedName)
 	return r.updateStatus(r.sync(log, req))
 }
 
 // If dataSourceRef is set (external populator), use an empty spec.Source field
-func (r PopulatorReconciler) prepare(syncRes *dataVolumeSyncResult) error {
+func (r *PopulatorReconciler) prepare(syncRes *dataVolumeSyncResult) error {
 	if !dvUsesVolumePopulator(syncRes.dv) {
 		return errors.Errorf("undefined population source")
 	}
+	// TODO - let's revisit this
+	syncRes.dv.Spec.Source = &cdiv1.DataVolumeSource{}
```

Comment:
@alromeros wdyt?

Reply:
@mhenriks what do you mean? Maybe adding a source for external populators?

Reply:
It just seemed awkward and unexpected we don't have to do anything in the pr.  I'll remove the comment